### PR TITLE
Loops: while and for

### PR DESCRIPTION
**Pull request by vahmelk99**

### CODE REVIEW

File: 1-js/02-first-steps/13-while-for/article.md [hunk 1]
```diff
@@ -284,114 +284,114 @@ if (i > 5) {
...
 ```js run no-beautify
 for (let i = 0; i < 3; i++) {
 
   for (let j = 0; j < 3; j++) {
 
-    let input = prompt(`Value at coords (${i},${j})`, '');
+    let input = prompt(`Արժեքը (${i},${j}) կոորդինատում`, '');
 
-    // what if we want to exit from here to Done (below)?
+    // ինչ եթե ցանկանում ենք դուրս գալ այստեղից, և անցնել Պատրաստ է (ներքև) տողին?
```
Comment (vahmelk99) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
    // ի՞նչ եթե ցանկանում ենք դուրս գալ այստեղից, և անցնել Պատրաստ է (ներքև) տողին
```

File: 1-js/02-first-steps/13-while-for/2-which-value-while/solution.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
+    Եվ վերջապես `i = 4` ավելացվում է դառնալով `5`, `while(5 < 5)` համեմատումը կլինի սխալ, և ցիկլը կավարտվի։ Այսպիսով `5`֊ը չի երևա էկրանին։
+2. **1֊ից մինչև 5**
 
     ```js run
     let i = 0;
     while (i++ < 5) alert( i );
     ```
 
-    The first value is again `i = 1`. The postfix form of `i++` increments `i` and then returns the *old* value, so the comparison `i++ < 5` will use `i = 0` (contrary to `++i < 5`).
+    Առաջին արժեքը կրկին `i = 1`։ `i++`֊ը աճեցնում է `i`֊ն, և ապա վերադարձնում *հին* արժեքը, այսպիսով `i++ < 5` համեմատության մեջ `i = 0` (ի տարբերություն `++i < 5`֊ի)։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/2-which-value-while/solution.md]:
```suggestion
    Առաջին արժեքը կրկին `i = 1` է։ `i++`֊ը աճեցնում է `i`֊ն, և ապա վերադարձնում *հին* արժեքը, այսպիսով `i++ < 5` համեմատության մեջ `i = 0` (ի տարբերություն `++i < 5`֊ի)։
```

File: 1-js/02-first-steps/13-while-for/5-replace-for-while/task.md [hunk 3]
```diff
@@ -2,13 +2,13 @@ importance: 5
 
 ---
 
-# Replace "for" with "while"
+# Փոխարինեք "for"֊ը "while"֊ով
 
-Rewrite the code changing the `for` loop to `while` without altering its behavior (the output should stay same).
+Վերաշարադրեք կոդը, փոխելով `for` ցիկլը `while`֊ով, առանց նրա կատարման արդյունքը փոխելու (ցուցադրվող արժեքները պետք է մնամ նույնը)։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/5-replace-for-while/task.md]:
```suggestion
Վերաշարադրեք կոդը, փոխելով `for` ցիկլը `while`֊ով, առանց նրա կատարման արդյունքը փոխելու (ցուցադրվող արժեքները պետք է մնան նույնը)։
```

File: 1-js/02-first-steps/13-while-for/6-repeat-until-correct/solution.md [hunk 4]
```diff
@@ -3,13 +3,13 @@
 let num;
 
 do {
-  num = prompt("Enter a number greater than 100?", 0);
+  num = prompt("Ներմուծեք 100֊ից բարձր թիվ", 0);
 } while (num <= 100 && num);
 ```
 
-The loop `do..while` repeats while both checks are truthy:
+`do..while` ցիկլը կրկնում է քանի դեռ երկու պայմաններն էլ ճշմարիտ չեն․
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/6-repeat-until-correct/solution.md]: This should actually be:
```suggestion
`do..while` ցիկլը կրկնում է քանի դեռ երկու պայմաններն էլ ճշմարիտ են․
```

File: 1-js/02-first-steps/13-while-for/7-list-primes/solution.md [hunk 5]
```diff
@@ -16,14 +16,14 @@ The code using a label:
 let n = 10;
 
 nextPrime:
-for (let i = 2; i <= n; i++) { // for each i...
+for (let i = 2; i <= n; i++) { // ամեն i֊ի համար...
 
-  for (let j = 2; j < i; j++) { // look for a divisor..
-    if (i % j == 0) continue nextPrime; // not a prime, go next i
+  for (let j = 2; j < i; j++) { // փնտրում է բաժանարար..
+    if (i % j == 0) continue nextPrime; // պերզ չէ, անցնում է հաջորդ i֊ին
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/7-list-primes/solution.md]:
```suggestion
    if (i % j == 0) continue nextPrime; // պարզ չէ, անցնում է հաջորդ i֊ին
```

File: 1-js/02-first-steps/13-while-for/7-list-primes/solution.md [hunk 6]
```diff
@@ -16,14 +16,14 @@ The code using a label:
 let n = 10;
 
 nextPrime:
-for (let i = 2; i <= n; i++) { // for each i...
+for (let i = 2; i <= n; i++) { // ամեն i֊ի համար...
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/7-list-primes/solution.md:19]: There are a few lines in the beginning of this file that are not translated. I think the pseudo-algorithm above can be left as is but if you want translate it I'm OK with that.

File: 1-js/02-first-steps/13-while-for/7-list-primes/task.md [hunk 7]
```diff
@@ -2,16 +2,16 @@ importance: 3
...
+`1`֊ից մեծ ամբողջ թիվը կոչվում է [պարզ](https://hy.wikipedia.org/wiki/Պարզ_թիվ), եթե ունի միայն երկու բաժանարար, դրանք են՝ `1`֊ը և հենց ինքը։
 
-In other words, `n > 1` is a prime if it can't be evenly divided by anything except `1` and `n`.
+Այլ կերպ ասած, `n > 1` պայմանին բավարարող `n`֊ը պարզ է, եթե այն հնարավոր չէ անմնացորդ բաժանել կամայական ամբողջ թվի վրա, բացի `1`֊ից և `n`֊ից։
 
-For example, `5` is a prime, because it cannot be divided without a remainder by `2`, `3` and `4`.
+Օրինակ, `5`֊ը պարզ է, քանի որ այն չի բաժանվում `2`֊ի, `3`֊ի և `4`֊ի։
 
-**Write the code which outputs prime numbers in the interval from `2` to `n`.**
+**Գրել ծրագրի, որը կտպի բոլոր պարզ թվերը `2`֊ից մինչև `n` հատվածում.**
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/7-list-primes/task.md]:
```suggestion
**Գրել ծրագիր, որը կտպի բոլոր պարզ թվերը `2`֊ից մինչև `n` հատվածում:**
```

**Review by bugron — CHANGES_REQUESTED** [review]
Overall looks good just a few comments to address. Besides those comments while you're resolving merge conflicts I'll get to reviewing the main article.

**Comment by vahmelk99** [comment]
Thanks for the review)

File: 1-js/02-first-steps/13-while-for/7-list-primes/solution.md [hunk 8]
```diff
@@ -1,29 +1,29 @@
...
-Let's use a nested loop:
+Եկեք օգտագործենք ներդրված ցիկլեր․
 
 ```js
-For each i in the interval {
-  check if i has a divisor from 1..i
-  if yes => the value is not a prime
-  if no => the value is a prime, show it
+Բոլոր i֊երի համար հատվածից {
+  ստուգու է, արդյոք i֊ն բաժանվում է ինչ֊որ թվի միչև ինքը
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/7-list-primes/solution.md]:
```suggestion
  ստուգում է, արդյոք i֊ն բաժանվում է ինչ֊որ թվի միչև ինքը
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 9]
```diff
@@ -1,54 +1,54 @@
-# Loops: while and for
+# Ցիկլեր․ while և for
 
-We often need to repeat actions.
+Հաճախ մենք կարիք ենք ունենում կրկնել ինչ֊որ գործողությունների հետթականություն։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Հաճախ մենք կարիք ենք ունենում կրկնել ինչ֊որ գործողությունների հերթականություն։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 10]
```diff
@@ -1,54 +1,54 @@
...
 
-For example, outputting goods from a list one after another or just running the same code for each number from 1 to 10.
+Օրինակ․ տպել իրար հետևից ցուցակի մեջ գտնվող ապրանքները, կամ ուղղակի աշխակատցնել նույն կոդը 1֊ից 10 բոլոր թվերի համար։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Օրինակ․ տպել իրար հետևից ցուցակի մեջ գտնվող ապրանքները, կամ ուղղակի աշխատեցնել նույն կոդը 1֊ից 10 բոլոր թվերի համար։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 11]
```diff
@@ -58,19 +58,19 @@ while (i) alert(i--);
...
 
 ```js
 do {
-  // loop body
+  // ցիկլի մարմին
 } while (condition);
 ```
 
-The loop will first execute the body, then check the condition, and, while it's truthy, execute it again and again.
+Ցիկլը սկզբում կաշխատացնի մարմնը, ապա կստուգի պայմանը, և քանի դեռ այն ճշմարիտ է, այն կաշխատացնի մարմինը նորից ու նորից։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Ցիկլը սկզբում կաշխատացնի մարմինը, ապա կստուգի պայմանը, և քանի դեռ այն ճշմարիտ է, այն կաշխատացնի մարմինը նորից ու նորից։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 12]
```diff
@@ -80,109 +80,109 @@ do {
...
 ```
 
-This form of syntax should only be used when you want the body of the loop to execute **at least once** regardless of the condition being truthy. Usually, the other form is preferred: `while(…) {…}`.
+Այս գրելաձևը պետք է օգտագործվի միայն այն դեպքում, երբ ցանկանում եք, որ ցիկլի մարմինը աշխատի **առնվազն մեկ անգամ** անկախ պայմանի ճշմարիտ կամ սխալական լինելու փաստից։ Սովորաբար մյուս տեսքը ավելի նախընտրելի է․ `while(…) {…}`։
 
-## The "for" loop
+## "for" ցիկլը
 
-The `for` loop is more complex, but it's also the most commonly used loop.
+`for` ցիլկը ավելի կոմպլեքս է, բայց այն ամենից շատ կիրառվող ցիկլն է։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
`for` ցիլկը ավելի բարդ է, բայց այն ամենից շատ կիրառվող ցիկլն է։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 13]
```diff
@@ -80,109 +80,109 @@ do {
...
 
 ```js
 for (begin; condition; step) {
-  // ... loop body ...
+  // ... ցիկլի մարմին ...
 }
 ```
 
-Let's learn the meaning of these parts by example. The loop below runs `alert(i)` for `i` from `0` up to (but not including) `3`:
+Եկեք ուսումնասիրենք այդ մասեիը օրինակի վրա։ Ներքևի ցիկլը կատարում է `alert(i)` հրամանը `i`֊ն `0`֊ից մինչև `3`֊ը ոչ ներառյալ:
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Եկեք ուսումնասիրենք այդ մասերը օրինակի վրա։ Ներքևի ցիկլը կատարում է `alert(i)` հրամանը `i`֊ն `0`֊ից մինչև `3`֊ը ոչ ներառյալ:
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 14]
```diff
@@ -80,109 +80,109 @@ do {
...
 
 ```js run
-for (let i = 0; i < 3; i++) { // shows 0, then 1, then 2
+for (let i = 0; i < 3; i++) { // ցույց կտա 0, հետո 1, հետո 2
   alert(i);
 }
 ```
 
-Let's examine the `for` statement part-by-part:
+Դիտարկենք `for`֊ը մաս֊առ֊մաս․
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Դիտարկենք `for`֊ը մաս առ մաս․
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 15]
```diff
@@ -80,109 +80,109 @@ do {
...
+| մաս  |          |                                                                            |
 |-------|----------|----------------------------------------------------------------------------|
-| begin | `let i = 0`    | Executes once upon entering the loop.                                      |
-| condition | `i < 3`| Checked before every loop iteration. If false, the loop stops.              |
-| body | `alert(i)`| Runs again and again while the condition is truthy.                         |
-| step| `i++`      | Executes after the body on each iteration. |
+| սկիզբ | `i = 0`    | Կատարվում է, երբ մտնում ենք ցիկլ։                                      |
+| պայման | `i < 3`| Ստուգվում է ցիկլի ամեն իտերացիայից առաջ։ Եթե այն սխալ է, ապա ցիկլը կանգնեցվում է։              |
+| մարմին | `alert(i)`| Աշխատում է այնքան ժամանակ, քանի դեռ պայմանը ճշմարիտ է։                         |
+| քայլ| `i++`      | Կատարվում է մարմնի կատարումից հետո, ամեն իտերացիայից ժամանակ։ |
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
| քայլ| `i++`      | Կատարվում է մարմնի կատարումից հետո, ամեն իտերացիայի ժամանակ։ |
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 16]
```diff
@@ -80,109 +80,109 @@ do {
...
-→ (if condition → run body and run step)
+Աշխատացնել *սկիզբ*
+→ (եթե *պայման* → կատարել *մարմին* և աշխատացնել *քայլ*)
+→ (եթե *պայման* → կատարել *մարմին* և աշխատացնել *քայլ*)
+→ (եթե *պայման* → կատարել *մարմին* և աշխատացնել *քայլ*)
 → ...
 ```
 
-That is, `begin` executes once, and then it iterates: after each `condition` test, `body` and `step` are executed.
+Այսինքն, `սկիզբ`֊ը կատարվում է մեկ անգամ, և հետո սկսվում են իտերացիաները․ ամեն `պայման`֊ի ստուգումից հետո, `մարմին`֊ը և `քայլ`֊ը կատարվում են։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Այսինքն՝ `սկիզբ`֊ը կատարվում է մեկ անգամ, և հետո սկսվում են իտերացիաները․ ամեն `պայման`֊ի ստուգումից հետո, `մարմին`֊ը և `քայլ`֊ը կատարվում են։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 17]
```diff
@@ -80,109 +80,109 @@ do {
...
 
-If you are new to loops, it could help to go back to the example and reproduce how it runs step-by-step on a piece of paper.
+Եթե ցիկլերի թեման նոր է ձեր համար, ապա օգտակար կլինի հետ գնալ օրինակներին և քայլ֊առ֊քայլ գրառել դրանց կատարման ընթացքները թղթի վրա։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Եթե ցիկլերի թեման նոր է ձեր համար, ապա օգտակար կլինի հետ գնալ օրինակներին և քայլ առ քայլ գրառել դրանց կատարման ընթացքները թղթի վրա։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 18]
```diff
@@ -192,32 +192,32 @@ for (; i < 3;) {
...
+## Ցիկլի ընդհատում (break)
 
-Normally, a loop exits when its condition becomes falsy.
+Սովորական դեպքերում ցիկլը դուրս է գալիս (ավարտվում է) այն ժամանակ, երբ նրա պայմանը դառնում է սխալական։
 
-But we can force the exit at any time using the special `break` directive.
+Բայց մենք կարող ենք հարկադրաբար (force) դուրս գալ ցիկլից, ամեն պահի, օգտագործելով հատուկ դիրեկտիվ `break`֊ը։
 
-For example, the loop below asks the user for a series of numbers, "breaking" when no number is entered:
+Օրինակ ներքևի ցիկլը օգտատիրոջից անընդհատ թվային մուտք է ուզում, "ընդհատվում" է, երբ ոչ թվային արժեք է մուտքագրվում․
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Օրինակ՝ ներքևի ցիկլը օգտատիրոջից անընդհատ թվային մուտք է ուզում, "ընդհատվում" է, երբ ոչ թվային արժեք է մուտքագրվում․
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 19]
```diff
@@ -226,35 +226,35 @@ while (true) {
...
+## Շարունակել հաջորդ իտերացիայից [#continue]
 
-The `continue` directive is a "lighter version" of `break`. It doesn't stop the whole loop. Instead, it stops the current iteration and forces the loop to start a new one (if the condition allows).
+`continue` դիրեկտիվը `break`֊ի ավելի "թեթև տարբերակն" է։ Այն չի կանգնեցնում ամբողջ ցիկլը, այլ կանգնեցնում է միայն ընթացիկ իտերացիան, և հարկադրում ցիկլին անցնել հաջորդին (եթե պայմանը ճշմարիտ է)։
 
-We can use it if we're done with the current iteration and would like to move on to the next one.
+Մենք կարող ենք օգտագործել այն, եթե այլևս կարիք չկա շարունակել այդ պահի իտերացիան և ցանկանում ենք անցնել հաջորդին։
 
-The loop below uses `continue` to output only odd values:
+Ներքև ցիկլը օգտագործում է `continue`֊ն ցուցադրելու միայն կենտ արժեքները․
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Ներքևի ցիկլը օգտագործում է `continue`֊ն՝ ցուցադրելու միայն կենտ արժեքները․
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 20]
```diff
@@ -266,15 +266,15 @@ for (let i = 0; i < 10; i++) {
...
+Տեխնիկական տեսանկյունից վերևում բերված երկու օրինակները ամբողջովին նույնն են։ Իհարկե, մենք կարող ենք շրջապատել մեր կոդի այդ հատվածը `if` բլոկով, `continue`֊ի փոխարեն։
 
-But as a side-effect, this created one more level of nesting (the `alert` call inside the curly braces). If the code inside of `if` is longer than a few lines, that may decrease the overall readability.
+Բայց մենք կստանանք լրացուցիչ ներդրված բլոկներ (`alert`֊ի կանչումը ձևավոր փակագծերի ներսում)։ Եթե `if`֊ի ներսի կոդը մի քանի տողից երկար է, ապա այն կարող է նվազեցնել ընդհանուր ծրագրի ընթեռնելիությունը։
 ````
 
-````warn header="No `break/continue` to the right side of '?'"
-Please note that syntax constructs that are not expressions cannot be used with the ternary operator `?`. In particular, directives such as `break/continue` aren't allowed there.
+````warn header="Ոչ մի `break/continue` '?'֊ի աջ կողմում"
+Ուշադրություն դարձրեք, որ սինտաքսիկ կոնդտրուկցիաները (syntax constructs) չեն հանդիիսանում արտահայտություններ (expressions) և չեն կարող օգտագործվել `?` օպերատորի հետ։ Մասնավոր դեպքում, դիրեկտիվները ինչպիսիք են `break/continue` թույլատրված չեն '?'֊ի հետ։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Ուշադրություն դարձրեք, որ սինտաքսիկ կոնդտրուկցիաները (syntax constructs) չեն հանդիսանում արտահայտություններ (expressions) և չեն կարող օգտագործվել `?` օպերատորի հետ։ Մասնավոր դեպքում, `break/continue` և նմանատիպ դիրեկտիվները թույլատրված չեն '?'֊ի հետ։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 21]
```diff
@@ -284,114 +284,114 @@ if (i > 5) {
...
 
-alert('Done!');
+alert('Պատրաստ է!');
 ```
 
-We need a way to stop the process if the user cancels the input.
+Մեզ պետք է ինչ֊որ ձև կանգնեցնել պրոցեսը, եթե օգտատերը չեղարկել է ներմուծման դաշտը։
 
-The ordinary `break` after `input` would only break the inner loop. That's not sufficient -- labels, come to the rescue!
+Ստանդարտ `break`֊ը `input`֊ից հետօ կկանգնեցնի միայն ներսի ցիկլը։ Դա բավարար չէ, և այստեղ է, որ պիտակները գալիս են օգնության։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Ստանդարտ `break`֊ը `input`֊ից հետո կկանգնեցնի միայն ներսի ցիկլը։ Դա բավարար չէ, և այստեղ է, որ պիտակները գալիս են օգնության։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 22]
```diff
@@ -284,114 +284,114 @@ if (i > 5) {
...
-    // do something with the value...
+    // արժեքի հետ ինչ֊որ գործողություն...
   }
 }
-alert('Done!');
+alert('Պատրաստ է!');
 ```
 
-In the code above, `break outer` looks upwards for the label named `outer` and breaks out of that loop.
+Վերևի ծրագրում `break outer`֊ը ման է գալիս իրեն ընդգրկող դրսի ցիկլ, որի պիտակը `outer` է, և կանգնեցնում այն։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Վերևի ծրագրում `break outer`֊ը փնտրում է իրեն ընդգրկող դրսի ցիկլը, որի պիտակը `outer` է, և կանգնեցնում այն։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 23]
```diff
@@ -284,114 +284,114 @@ if (i > 5) {
...
 
-So the control goes straight from `(*)` to `alert('Done!')`.
+Այդպիսով կատարում `(*)`֊ից գնում է միանգամից `alert('Done!')`։
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Այդպիսով կատարումը `(*)`֊ից անցնում է միանգամից `alert('Done!')`։
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 24]
```diff
@@ -284,114 +284,114 @@ if (i > 5) {
...
 
-We can also move the label onto a separate line:
+Մենք նաև կարոց ենք պիտակները տեղադրել ուրիշ տողում․
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
Մենք նաև կարող ենք պիտակները տեղադրել ուրիշ տողում․
```

File: 1-js/02-first-steps/13-while-for/article.md [hunk 25]
```diff
@@ -284,114 +284,114 @@ if (i > 5) {
...
+Օրինակ, անհնար է անել այսպիրի հնարք․
 ```js
-break label; // jump to the label below (doesn't work)
+break label; // ցատկ ներքևի պիտակին (չի աշխատի)
 
 label: for (...)
 ```
 
-A `break` directive must be inside a code block. Technically, any labelled code block will do, e.g.:
+`break` դիրեկտիվը պետք է լինի պիտակը պարունակող բլոկի ներսում։ Տեխնիկորեն կամայական պիտակավորված բլոկի համար կարող ենք անել հետևյալը․
```
Comment (bugron) [inline 1-js/02-first-steps/13-while-for/article.md]:
```suggestion
`break` դիրեկտիվը պետք է լինի պիտակը պարունակող բլոկի ներսում։ Տեխնիկապես կամայական պիտակավորված բլոկի համար կարող ենք անել հետևյալը․
```

**Review by bugron — CHANGES_REQUESTED** [review]
Let's address a few more comments and we're good to go.

**Comment by bugron** [comment]
This is awesome! Thanks @vahmelk99 for all your hard work. It is my pleasure to collaborate with you. Way to go! Merged.

**Pull request merged**